lib/core/Canvas.js

```diff
@@ -210,7 +213,7 @@ Canvas.prototype._clear = function() {
  * @returns {Snap<SVGGroup>}
  */
 Canvas.prototype.getDefaultLayer = function() {
-  return this.getLayer(BASE_LAYER);
+  return this.getLayer(BASE_LAYER, 0);
```

Comment:
Is this a backwards compatible change?

Previously it was possible to create layers `A`, `Base` and `C` in this order by calling:

```
getLayer('A');
getBaseLayer();
getLayer('C');
```

Reply:
Yes, this works as expected.